packages/eslint-plugin/rules/data-no-store-string-literals.js

```diff
@@ -85,6 +115,82 @@ function collectAllNodesFromObjectPropertyFunctionCalls( context, node ) {
 	return possibleCallExpressionNodes;
 }
 
+function getSuggest( context, callNode ) {
+	return [
+		{
+			desc:
+				'Replace literal with store definition. Import store if neccessary.',
+			fix: ( fixer ) => getFixes( fixer, context, callNode ),
+		},
+	];
+}
+
+function getFixes( fixer, context, callNode ) {
+	const storeName = callNode.arguments[ 0 ].value;
+	const storeDefinitions = {
+		core: {
+			import: '@wordpress/core-data',
+			variable: 'coreStore',
+		},
+	};
+	let storeDefinition = storeDefinitions[ storeName ];
+	if ( ! storeDefinition && storeName.startsWith( 'core/' ) ) {
+		const storeNameWithoutCore = storeName.substring( 5 );
+		storeDefinition = {
+			import: `@wordpress/${ storeNameWithoutCore }`,
+			variable: storeNameToVariableNames( storeNameWithoutCore ),
+		};
+	}
+	if ( ! storeDefinition ) {
+		return null;
+	}
+	const { variable: variableName, import: importName } = storeDefinition;
+
+	const fixes = [
+		fixer.replaceText( callNode.arguments[ 0 ], variableName ),
+	];
+
+	const imports = context
+		.getAncestors()[ 0 ]
+		.body.filter( ( n ) => n.type === 'ImportDeclaration' );
```

Comment:
Nit, should it be `node` here?

```suggestion
		.body.filter( ( node ) => node.type === 'ImportDeclaration' );
```